[2024-04-18T04:14:59.928+0000] {processor.py:161} INFO - Started process (PID=44000) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:14:59.929+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:14:59.931+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:14:59.930+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:15:00.444+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:15:00.430+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 15, in <module>
    python_callable = ExtractApi
                      ^^^^^^^^^^
NameError: name 'ExtractApi' is not defined
[2024-04-18T04:15:00.457+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:15:00.483+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.563 seconds
[2024-04-18T04:15:05.115+0000] {processor.py:161} INFO - Started process (PID=44006) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:15:05.118+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:15:05.121+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:15:05.120+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:15:05.656+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:15:05.641+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 15, in <module>
    python_callable = ExtractApi
                      ^^^^^^^^^^
NameError: name 'ExtractApi' is not defined
[2024-04-18T04:15:05.665+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:15:05.692+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.590 seconds
[2024-04-18T04:15:35.922+0000] {processor.py:161} INFO - Started process (PID=44072) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:15:35.924+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:15:35.926+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:15:35.925+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:15:36.434+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:15:36.424+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 15, in <module>
    python_callable = ExtractApi
                      ^^^^^^^^^^
NameError: name 'ExtractApi' is not defined
[2024-04-18T04:15:36.441+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:15:36.580+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.667 seconds
[2024-04-18T04:16:01.495+0000] {processor.py:161} INFO - Started process (PID=44134) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:16:01.496+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:16:01.498+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:16:01.497+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:16:02.008+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:16:01.996+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 15, in <module>
    python_callable = ExtractApi
                      ^^^^^^^^^^
NameError: name 'ExtractApi' is not defined
[2024-04-18T04:16:02.016+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:16:02.050+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.564 seconds
[2024-04-18T04:16:32.666+0000] {processor.py:161} INFO - Started process (PID=44200) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:16:32.667+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:16:32.669+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:16:32.668+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:16:33.235+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:16:33.223+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 15, in <module>
    python_callable = ExtractApi
                      ^^^^^^^^^^
NameError: name 'ExtractApi' is not defined
[2024-04-18T04:16:33.244+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:16:33.275+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.618 seconds
[2024-04-18T04:17:03.574+0000] {processor.py:161} INFO - Started process (PID=44258) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:17:03.576+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:17:03.578+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:17:03.577+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:17:04.139+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:17:04.127+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 15, in <module>
    python_callable = ExtractApi
                      ^^^^^^^^^^
NameError: name 'ExtractApi' is not defined
[2024-04-18T04:17:04.148+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:17:04.173+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.616 seconds
[2024-04-18T04:17:29.853+0000] {processor.py:161} INFO - Started process (PID=44296) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:17:29.855+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:17:29.857+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:17:29.856+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:17:29.889+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:17:29.887+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 7
    from AggData import ExtractApi, 
                                    ^
SyntaxError: trailing comma not allowed without surrounding parentheses
[2024-04-18T04:17:29.890+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:17:29.920+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.085 seconds
[2024-04-18T04:18:00.186+0000] {processor.py:161} INFO - Started process (PID=44361) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:18:00.188+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:18:00.189+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:18:00.189+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:18:00.224+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:18:00.222+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 7
    from AggData import ExtractApi, 
                                    ^
SyntaxError: trailing comma not allowed without surrounding parentheses
[2024-04-18T04:18:00.225+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:18:00.268+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.090 seconds
[2024-04-18T04:18:30.464+0000] {processor.py:161} INFO - Started process (PID=44424) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:18:30.466+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:18:30.467+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:18:30.467+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:18:30.517+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:18:30.515+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 7
    from AggData import ExtractApi, 
                                    ^
SyntaxError: trailing comma not allowed without surrounding parentheses
[2024-04-18T04:18:30.518+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:18:30.643+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.188 seconds
[2024-04-18T04:19:00.722+0000] {processor.py:161} INFO - Started process (PID=44488) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:19:00.724+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:19:00.725+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:19:00.725+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:19:00.761+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:19:00.759+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 7
    from AggData import ExtractApi, 
                                    ^
SyntaxError: trailing comma not allowed without surrounding parentheses
[2024-04-18T04:19:00.762+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:19:00.894+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.182 seconds
[2024-04-18T04:19:31.169+0000] {processor.py:161} INFO - Started process (PID=44553) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:19:31.171+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:19:31.173+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:19:31.172+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:19:31.212+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:19:31.211+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 7
    from AggData import ExtractApi, 
                                    ^
SyntaxError: trailing comma not allowed without surrounding parentheses
[2024-04-18T04:19:31.214+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:19:31.280+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.121 seconds
[2024-04-18T04:19:56.444+0000] {processor.py:161} INFO - Started process (PID=44601) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:19:56.446+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:19:56.448+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:19:56.447+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:19:56.967+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:19:56.958+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 17, in <module>
    python_callable = AggData.ExtractApi
                      ^^^^^^^
NameError: name 'AggData' is not defined
[2024-04-18T04:19:56.973+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:19:57.004+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.570 seconds
[2024-04-18T04:20:13.556+0000] {processor.py:161} INFO - Started process (PID=44663) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:20:13.558+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:20:13.560+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:20:13.559+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:20:14.101+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:20:14.090+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 20, in <module>
    python_callable = AggData.ExtractApi
                      ^^^^^^^
NameError: name 'AggData' is not defined
[2024-04-18T04:20:14.106+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:20:14.134+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.588 seconds
[2024-04-18T04:20:44.231+0000] {processor.py:161} INFO - Started process (PID=44729) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:20:44.233+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:20:44.235+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:20:44.235+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:20:44.733+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:20:44.724+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 20, in <module>
    python_callable = AggData.ExtractApi
                      ^^^^^^^
NameError: name 'AggData' is not defined
[2024-04-18T04:20:44.739+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:20:44.767+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.547 seconds
[2024-04-18T04:21:12.746+0000] {processor.py:161} INFO - Started process (PID=44793) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:12.747+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:21:12.748+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:21:12.748+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:13.277+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:21:13.266+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 20, in <module>
    python_callable = AggData.ExtractApi
                      ^^^^^^^
NameError: name 'AggData' is not defined
[2024-04-18T04:21:13.287+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:13.321+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.585 seconds
[2024-04-18T04:21:17.648+0000] {processor.py:161} INFO - Started process (PID=44799) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:17.650+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:21:17.652+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:21:17.652+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:18.247+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:21:18.233+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 14, in <module>
    from funcoes import minha_funcao
ModuleNotFoundError: No module named 'funcoes'
[2024-04-18T04:21:18.250+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:18.288+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.656 seconds
[2024-04-18T04:21:32.548+0000] {processor.py:161} INFO - Started process (PID=44818) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:32.549+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:21:32.551+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:21:32.550+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:33.084+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:21:33.069+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 23, in <module>
    python_callable = AggData.ExtractApi
                      ^^^^^^^
NameError: name 'AggData' is not defined
[2024-04-18T04:21:33.092+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:33.124+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.586 seconds
[2024-04-18T04:21:37.252+0000] {processor.py:161} INFO - Started process (PID=44838) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:37.254+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:21:37.255+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:21:37.255+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:37.758+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:21:37.748+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 29, in <module>
    python_callable = ValidateData
                      ^^^^^^^^^^^^
NameError: name 'ValidateData' is not defined
[2024-04-18T04:21:37.766+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:37.794+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.553 seconds
[2024-04-18T04:21:58.236+0000] {processor.py:161} INFO - Started process (PID=44871) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:58.238+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:21:58.239+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:21:58.239+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:58.737+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:21:58.725+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:21:58.746+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:21:58.771+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.549 seconds
[2024-04-18T04:22:13.473+0000] {processor.py:161} INFO - Started process (PID=44929) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:22:13.475+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:22:13.477+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:22:13.476+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:22:14.061+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:22:14.036+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:22:14.077+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:22:14.115+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.652 seconds
[2024-04-18T04:22:30.238+0000] {processor.py:161} INFO - Started process (PID=44950) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:22:30.240+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:22:30.241+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:22:30.241+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:22:30.788+0000] {processor.py:840} INFO - DAG(s) 'dagExtract' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:22:30.925+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:22:30.925+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-18T04:22:30.968+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:22:30.967+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 03:30:00+00:00, run_after=2024-04-18 04:30:00+00:00
[2024-04-18T04:22:31.007+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.780 seconds
[2024-04-18T04:22:37.173+0000] {processor.py:161} INFO - Started process (PID=44970) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:22:37.175+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:22:37.176+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:22:37.176+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:22:37.649+0000] {processor.py:840} INFO - DAG(s) 'dagExtract' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:22:37.670+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:22:37.669+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-18T04:22:37.706+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:22:37.706+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 03:30:00+00:00, run_after=2024-04-18 04:30:00+00:00
[2024-04-18T04:22:37.737+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.573 seconds
[2024-04-18T04:23:04.422+0000] {processor.py:161} INFO - Started process (PID=45023) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:23:04.423+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:23:04.425+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:23:04.424+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:23:04.973+0000] {processor.py:840} INFO - DAG(s) 'dagExtract', 'dagOrchestration' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:23:05.154+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:23:05.153+0000] {override.py:1829} INFO - Created Permission View: can read on DAG:dagOrchestration
[2024-04-18T04:23:05.172+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:23:05.171+0000] {override.py:1829} INFO - Created Permission View: can edit on DAG:dagOrchestration
[2024-04-18T04:23:05.185+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:23:05.184+0000] {override.py:1829} INFO - Created Permission View: can delete on DAG:dagOrchestration
[2024-04-18T04:23:05.186+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:23:05.185+0000] {dag.py:3096} INFO - Sync 2 DAGs
[2024-04-18T04:23:05.204+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:23:05.204+0000] {dag.py:3118} INFO - Creating ORM DAG for dagOrchestration
[2024-04-18T04:23:05.229+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:23:05.229+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 03:30:00+00:00, run_after=2024-04-18 04:30:00+00:00
[2024-04-18T04:23:05.236+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:23:05.236+0000] {dag.py:3954} INFO - Setting next_dagrun for dagOrchestration to 2024-04-17 00:00:00+00:00, run_after=2024-04-18 00:00:00+00:00
[2024-04-18T04:23:05.269+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.856 seconds
[2024-04-18T04:23:35.986+0000] {processor.py:161} INFO - Started process (PID=45103) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:23:35.987+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:23:35.989+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:23:35.988+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:23:36.529+0000] {processor.py:840} INFO - DAG(s) 'dagExtract', 'dagOrchestration' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:23:36.557+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:23:36.557+0000] {dag.py:3096} INFO - Sync 2 DAGs
[2024-04-18T04:23:36.592+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:23:36.592+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 03:30:00+00:00, run_after=2024-04-18 04:30:00+00:00
[2024-04-18T04:23:36.602+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:23:36.602+0000] {dag.py:3954} INFO - Setting next_dagrun for dagOrchestration to 2024-04-18 00:00:00+00:00, run_after=2024-04-19 00:00:00+00:00
[2024-04-18T04:23:36.627+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.650 seconds
[2024-04-18T04:24:06.794+0000] {processor.py:161} INFO - Started process (PID=45171) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:24:06.796+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:24:06.798+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:24:06.797+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:24:07.308+0000] {processor.py:840} INFO - DAG(s) 'dagOrchestration', 'dagExtract' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:24:07.435+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:24:07.435+0000] {dag.py:3096} INFO - Sync 2 DAGs
[2024-04-18T04:24:07.463+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:24:07.463+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 03:30:00+00:00, run_after=2024-04-18 04:30:00+00:00
[2024-04-18T04:24:07.469+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:24:07.469+0000] {dag.py:3954} INFO - Setting next_dagrun for dagOrchestration to 2024-04-18 00:00:00+00:00, run_after=2024-04-19 00:00:00+00:00
[2024-04-18T04:24:07.495+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.709 seconds
[2024-04-18T04:24:38.180+0000] {processor.py:161} INFO - Started process (PID=45237) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:24:38.182+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:24:38.183+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:24:38.183+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:24:38.696+0000] {processor.py:840} INFO - DAG(s) 'dagOrchestration', 'dagExtract' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:24:38.732+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:24:38.732+0000] {dag.py:3096} INFO - Sync 2 DAGs
[2024-04-18T04:24:38.766+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:24:38.766+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 03:30:00+00:00, run_after=2024-04-18 04:30:00+00:00
[2024-04-18T04:24:38.775+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:24:38.774+0000] {dag.py:3954} INFO - Setting next_dagrun for dagOrchestration to 2024-04-18 00:00:00+00:00, run_after=2024-04-19 00:00:00+00:00
[2024-04-18T04:24:38.799+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.629 seconds
[2024-04-18T04:25:08.934+0000] {processor.py:161} INFO - Started process (PID=45304) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:25:08.936+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:25:08.937+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:25:08.937+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:25:09.431+0000] {processor.py:840} INFO - DAG(s) 'dagOrchestration', 'dagExtract' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:25:09.458+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:25:09.458+0000] {dag.py:3096} INFO - Sync 2 DAGs
[2024-04-18T04:25:09.488+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:25:09.487+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 03:30:00+00:00, run_after=2024-04-18 04:30:00+00:00
[2024-04-18T04:25:09.495+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:25:09.495+0000] {dag.py:3954} INFO - Setting next_dagrun for dagOrchestration to 2024-04-18 00:00:00+00:00, run_after=2024-04-19 00:00:00+00:00
[2024-04-18T04:25:09.516+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.591 seconds
[2024-04-18T04:25:39.602+0000] {processor.py:161} INFO - Started process (PID=45372) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:25:39.603+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:25:39.604+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:25:39.604+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:25:40.082+0000] {processor.py:840} INFO - DAG(s) 'dagExtract', 'dagOrchestration' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:25:40.214+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:25:40.214+0000] {dag.py:3096} INFO - Sync 2 DAGs
[2024-04-18T04:25:40.246+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:25:40.246+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 03:30:00+00:00, run_after=2024-04-18 04:30:00+00:00
[2024-04-18T04:25:40.252+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:25:40.252+0000] {dag.py:3954} INFO - Setting next_dagrun for dagOrchestration to 2024-04-18 00:00:00+00:00, run_after=2024-04-19 00:00:00+00:00
[2024-04-18T04:25:40.300+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.708 seconds
[2024-04-18T04:26:10.903+0000] {processor.py:161} INFO - Started process (PID=45438) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:26:10.905+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:26:10.906+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:26:10.906+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:26:11.399+0000] {processor.py:840} INFO - DAG(s) 'dagOrchestration', 'dagExtract' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:26:11.429+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:26:11.429+0000] {dag.py:3096} INFO - Sync 2 DAGs
[2024-04-18T04:26:11.463+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:26:11.463+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 03:30:00+00:00, run_after=2024-04-18 04:30:00+00:00
[2024-04-18T04:26:11.471+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:26:11.471+0000] {dag.py:3954} INFO - Setting next_dagrun for dagOrchestration to 2024-04-18 00:00:00+00:00, run_after=2024-04-19 00:00:00+00:00
[2024-04-18T04:26:11.493+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.599 seconds
[2024-04-18T04:26:41.650+0000] {processor.py:161} INFO - Started process (PID=45506) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:26:41.651+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:26:41.653+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:26:41.652+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:26:42.110+0000] {processor.py:840} INFO - DAG(s) 'dagOrchestration', 'dagExtract' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:26:42.136+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:26:42.135+0000] {dag.py:3096} INFO - Sync 2 DAGs
[2024-04-18T04:26:42.166+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:26:42.166+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 03:30:00+00:00, run_after=2024-04-18 04:30:00+00:00
[2024-04-18T04:26:42.178+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:26:42.177+0000] {dag.py:3954} INFO - Setting next_dagrun for dagOrchestration to 2024-04-18 00:00:00+00:00, run_after=2024-04-19 00:00:00+00:00
[2024-04-18T04:26:42.514+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.876 seconds
[2024-04-18T04:27:12.784+0000] {processor.py:161} INFO - Started process (PID=45574) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:27:12.786+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:27:12.788+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:27:12.787+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:27:13.307+0000] {processor.py:840} INFO - DAG(s) 'dagExtract', 'dagOrchestration' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:27:13.477+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:27:13.477+0000] {dag.py:3096} INFO - Sync 2 DAGs
[2024-04-18T04:27:13.512+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:27:13.511+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 03:30:00+00:00, run_after=2024-04-18 04:30:00+00:00
[2024-04-18T04:27:13.519+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:27:13.519+0000] {dag.py:3954} INFO - Setting next_dagrun for dagOrchestration to 2024-04-18 00:00:00+00:00, run_after=2024-04-19 00:00:00+00:00
[2024-04-18T04:27:13.614+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.841 seconds
[2024-04-18T04:27:44.332+0000] {processor.py:161} INFO - Started process (PID=45640) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:27:44.334+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:27:44.336+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:27:44.335+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:27:44.872+0000] {processor.py:840} INFO - DAG(s) 'dagExtract', 'dagOrchestration' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:27:44.905+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:27:44.905+0000] {dag.py:3096} INFO - Sync 2 DAGs
[2024-04-18T04:27:44.942+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:27:44.942+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 03:30:00+00:00, run_after=2024-04-18 04:30:00+00:00
[2024-04-18T04:27:44.950+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:27:44.950+0000] {dag.py:3954} INFO - Setting next_dagrun for dagOrchestration to 2024-04-18 00:00:00+00:00, run_after=2024-04-19 00:00:00+00:00
[2024-04-18T04:27:44.978+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.655 seconds
[2024-04-18T04:28:15.050+0000] {processor.py:161} INFO - Started process (PID=45709) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:28:15.051+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:28:15.052+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:15.052+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:28:15.512+0000] {processor.py:840} INFO - DAG(s) 'dagExtract', 'dagOrchestration' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:28:15.538+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:15.538+0000] {dag.py:3096} INFO - Sync 2 DAGs
[2024-04-18T04:28:15.575+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:15.575+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 03:30:00+00:00, run_after=2024-04-18 04:30:00+00:00
[2024-04-18T04:28:15.583+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:15.583+0000] {dag.py:3954} INFO - Setting next_dagrun for dagOrchestration to 2024-04-18 00:00:00+00:00, run_after=2024-04-19 00:00:00+00:00
[2024-04-18T04:28:15.607+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.565 seconds
[2024-04-18T04:28:46.396+0000] {processor.py:161} INFO - Started process (PID=45775) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:28:46.397+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:28:46.399+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:46.399+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:28:46.966+0000] {processor.py:840} INFO - DAG(s) 'dagExtract' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:28:47.294+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:47.294+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-18T04:28:47.322+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:47.322+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 00:00:00+00:00, run_after=2024-04-19 00:00:00+00:00
[2024-04-18T04:28:47.512+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 1.126 seconds
[2024-04-18T04:28:48.738+0000] {processor.py:161} INFO - Started process (PID=45785) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:28:48.739+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:28:48.741+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:48.740+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:28:49.206+0000] {processor.py:840} INFO - DAG(s) 'dagExtract', 'dagExtractApi' retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:28:49.530+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:49.529+0000] {override.py:1829} INFO - Created Permission View: can read on DAG:dagExtractApi
[2024-04-18T04:28:49.547+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:49.546+0000] {override.py:1829} INFO - Created Permission View: can edit on DAG:dagExtractApi
[2024-04-18T04:28:49.559+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:49.558+0000] {override.py:1829} INFO - Created Permission View: can delete on DAG:dagExtractApi
[2024-04-18T04:28:49.560+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:49.559+0000] {dag.py:3096} INFO - Sync 2 DAGs
[2024-04-18T04:28:49.573+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:49.573+0000] {dag.py:3118} INFO - Creating ORM DAG for dagExtractApi
[2024-04-18T04:28:49.587+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:49.586+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtract to 2024-04-18 03:30:00+00:00, run_after=2024-04-18 04:30:00+00:00
[2024-04-18T04:28:49.592+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:28:49.592+0000] {dag.py:3954} INFO - Setting next_dagrun for dagExtractApi to 2024-04-17 00:00:00+00:00, run_after=2024-04-18 00:00:00+00:00
[2024-04-18T04:28:49.615+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.886 seconds
[2024-04-18T04:29:19.604+0000] {processor.py:161} INFO - Started process (PID=45851) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:29:19.606+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:29:19.608+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:29:19.607+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:29:20.304+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:29:20.292+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:29:20.312+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:29:20.339+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.745 seconds
[2024-04-18T04:29:34.993+0000] {processor.py:161} INFO - Started process (PID=45884) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:29:34.995+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:29:34.997+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:29:34.996+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:29:35.573+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:29:35.556+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:29:35.588+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:29:35.620+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.636 seconds
[2024-04-18T04:29:51.742+0000] {processor.py:161} INFO - Started process (PID=45922) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:29:51.743+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:29:51.745+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:29:51.745+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:29:52.313+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:29:52.299+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:29:52.326+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:29:52.357+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.624 seconds
[2024-04-18T04:30:22.561+0000] {processor.py:161} INFO - Started process (PID=45988) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:30:22.564+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:30:22.566+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:30:22.565+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:30:23.145+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:30:23.132+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:30:23.154+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:30:23.184+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.640 seconds
[2024-04-18T04:30:53.379+0000] {processor.py:161} INFO - Started process (PID=46057) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:30:53.381+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:30:53.382+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:30:53.382+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:30:53.936+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:30:53.921+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:30:53.946+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:30:53.977+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.610 seconds
[2024-04-18T04:31:24.550+0000] {processor.py:161} INFO - Started process (PID=46127) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:31:24.552+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:31:24.553+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:31:24.553+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:31:25.056+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:31:25.041+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:31:25.064+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:31:25.088+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.553 seconds
[2024-04-18T04:31:55.347+0000] {processor.py:161} INFO - Started process (PID=46193) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:31:55.349+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:31:55.352+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:31:55.351+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:31:55.917+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:31:55.903+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:31:55.926+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:31:55.957+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.622 seconds
[2024-04-18T04:32:26.331+0000] {processor.py:161} INFO - Started process (PID=46264) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:32:26.332+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:32:26.333+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:32:26.333+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:32:26.866+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:32:26.851+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 14, in <module>
    from ExtractApi import *
  File "/opt/airflow/dags/ExtractApi.py", line 15, in <module>
    from ExtractUtils import *
ModuleNotFoundError: No module named 'ExtractUtils'
[2024-04-18T04:32:26.870+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:32:27.032+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.710 seconds
[2024-04-18T04:32:57.170+0000] {processor.py:161} INFO - Started process (PID=46334) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:32:57.172+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:32:57.174+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:32:57.173+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:32:57.660+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:32:57.645+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 14, in <module>
    from ExtractApi import *
  File "/opt/airflow/dags/ExtractApi.py", line 15, in <module>
    from ExtractUtils import *
ModuleNotFoundError: No module named 'ExtractUtils'
[2024-04-18T04:32:57.665+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:32:57.697+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.536 seconds
[2024-04-18T04:33:28.127+0000] {processor.py:161} INFO - Started process (PID=46402) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:33:28.129+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:33:28.130+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:33:28.130+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:33:28.642+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:33:28.628+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 14, in <module>
    from ExtractApi import *
  File "/opt/airflow/dags/ExtractApi.py", line 15, in <module>
    from ExtractUtils import *
ModuleNotFoundError: No module named 'ExtractUtils'
[2024-04-18T04:33:28.646+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:33:28.676+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.559 seconds
[2024-04-18T04:33:58.802+0000] {processor.py:161} INFO - Started process (PID=46466) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:33:58.803+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:33:58.804+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:33:58.804+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:33:59.306+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:33:59.289+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 14, in <module>
    from ExtractApi import *
  File "/opt/airflow/dags/ExtractApi.py", line 15, in <module>
    from ExtractUtils import *
ModuleNotFoundError: No module named 'ExtractUtils'
[2024-04-18T04:33:59.311+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:33:59.533+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.739 seconds
[2024-04-18T04:34:30.410+0000] {processor.py:161} INFO - Started process (PID=46541) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:34:30.413+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:34:30.414+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:34:30.414+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:34:31.112+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:34:31.094+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 14, in <module>
    from ExtractApi import *
  File "/opt/airflow/dags/ExtractApi.py", line 15, in <module>
    from ExtractUtils import *
ModuleNotFoundError: No module named 'ExtractUtils'
[2024-04-18T04:34:31.115+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:34:31.364+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.963 seconds
[2024-04-18T04:35:02.318+0000] {processor.py:161} INFO - Started process (PID=46614) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:35:02.319+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:35:02.321+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:35:02.320+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:35:02.984+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:35:02.973+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:35:02.992+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:35:03.013+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.708 seconds
[2024-04-18T04:35:33.906+0000] {processor.py:161} INFO - Started process (PID=46681) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:35:33.908+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:35:33.910+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:35:33.909+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:35:34.666+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:35:34.654+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:35:34.675+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:35:34.703+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.808 seconds
[2024-04-18T04:36:05.313+0000] {processor.py:161} INFO - Started process (PID=46748) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:36:05.314+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:36:05.316+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:36:05.315+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:36:06.056+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:36:06.039+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:36:06.066+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:36:06.088+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.784 seconds
[2024-04-18T04:36:36.526+0000] {processor.py:161} INFO - Started process (PID=46820) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:36:36.527+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:36:36.529+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:36:36.528+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:36:37.286+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:36:37.273+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:36:37.295+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:36:37.324+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.807 seconds
[2024-04-18T04:37:07.562+0000] {processor.py:161} INFO - Started process (PID=46888) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:37:07.564+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:37:07.565+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:37:07.565+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:37:08.244+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:37:08.234+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:37:08.251+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:37:08.277+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.724 seconds
[2024-04-18T04:37:38.885+0000] {processor.py:161} INFO - Started process (PID=46954) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:37:38.887+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:37:38.889+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:37:38.888+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:37:39.639+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:37:39.624+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:37:39.649+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:37:39.675+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.798 seconds
[2024-04-18T04:38:10.419+0000] {processor.py:161} INFO - Started process (PID=47022) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:38:10.421+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:38:10.423+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:38:10.422+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:38:11.235+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:38:11.220+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:38:11.245+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:38:11.279+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.870 seconds
[2024-04-18T04:38:41.456+0000] {processor.py:161} INFO - Started process (PID=47088) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:38:41.458+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:38:41.460+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:38:41.459+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:38:42.211+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:38:42.188+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:38:42.223+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:38:42.252+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.805 seconds
[2024-04-18T04:39:12.368+0000] {processor.py:161} INFO - Started process (PID=47154) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:39:12.371+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:39:12.373+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:39:12.372+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:39:13.174+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:39:13.151+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:39:13.183+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:39:13.206+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.849 seconds
[2024-04-18T04:39:43.748+0000] {processor.py:161} INFO - Started process (PID=47220) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:39:43.750+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:39:43.752+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:39:43.751+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:39:44.492+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:39:44.479+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:39:44.502+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:39:44.528+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.794 seconds
[2024-04-18T04:40:14.719+0000] {processor.py:161} INFO - Started process (PID=47288) to work on /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:40:14.721+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/DagOrchestration.py for tasks to queue
[2024-04-18T04:40:14.722+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:40:14.722+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:40:15.520+0000] {logging_mixin.py:188} INFO - [2024-04-18T04:40:15.507+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/DagOrchestration.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/DagOrchestration.py", line 52, in <module>
    python_callable = ReadBronze
                      ^^^^^^^^^^
NameError: name 'ReadBronze' is not defined
[2024-04-18T04:40:15.530+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/DagOrchestration.py
[2024-04-18T04:40:15.613+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/DagOrchestration.py took 0.904 seconds
